apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -105,6 +114,9 @@ class SectionProgress extends Component {
                 scriptData={scriptData}
                 studentLevelProgress={studentLevelProgress}
               />
+              <SummaryViewLegend
```

Comment:
I noticed this got changed in the other PR too, intentional?

Reply:
My guess is it floated around when I was fixing a merge conflict. Since we want the legend at the bottom of the view, I moved it.